core/admin/mailu/configuration.py

```diff
@@ -144,6 +145,8 @@ def init_app(self, app):
 
         # automatically set the sqlalchemy string
         if self.config['DB_FLAVOR']:
+            if self.config['DB_PORT'] and self.config['DB_FLAVOR'] != 'sqlite':
+                self.config['DB_HOST'] = f'{self.config["DB_HOST"]}:{str(self.config["DB_PORT"])}'
```

Comment:
you don't need to use `str()` inside the `{}`
This would be even shorter:
`self.config['DB_HOST'] += f':{self.config["DB_PORT"]}'`